doc/developers_guide/developers_guide.md

```diff
@@ -1,5 +1,11 @@
 # Developers Guide
 
+## Running PK locally
+
+You'll need to get the latest CLI version of project keeper, you can get it in the repository. Use this to update the project.
+Run the project-keeper fix command like this:
+`java -jar /path-to-pj-jar/project-keeper-cli-5.2.3.jar fix`
```

Comment:
```suggestion
Run PK locally in this repo using the generated shell script:
`.github/workflows/project-keeper.sh fix`
This will automatically download and run the correct PK version.
```